OpenDreamRuntime/DreamConnection.cs

```diff
@@ -210,30 +213,29 @@ public void HandleMsgPromptResponse(MsgPromptResponse message)
 
 
         public void OutputDreamValue(DreamValue value) {
-            if (value.Type == DreamValue.DreamValueType.DreamObject) {
-                DreamObject outputObject = value.GetValueAsDreamObject();
-
+            if (value.Type == DreamValue.DreamValueType.DreamObject && value.TryGetValueAsDreamObject(out var outputObject)) {
```

Comment:
`TryGetValueAsDreamObject()` already does the first check for you.
```suggestion
            if (value.TryGetValueAsDreamObject(out var outputObject)) {
```